### ISSUE TITLE
Plotly doesn't work without internet connection

### ISSUE THREAD

**Issue by Simbaclaws** [plotly/angular-plotly.js#153]
Hi there,

Apparently when I try to use plotly from a hybrid mobile app using ionic, I found out that on a actual iphone device this is giving an error message in the console saying:

Error loading plotly.js library from https://cdn.plot.ly/plotly-basic-latest.min.js. Timeout.

The problem is that this happens because I don't have an internet connection on the iPhone.
However I need my app to be able to work offline.

Apparently this .min.js is always loaded from a cdn for some reason.
How do I use these files locally (compiled) instead of over a cdn?

I'm using:
```
import * as PlotlyJS from 'plotly.js/dist/plotly.min.js';
import { PlotlyModule } from 'angular-plotly.js';

PlotlyModule.plotlyjs = PlotlyJS;
```
in my components.module.ts and

```
"projects": {
    "app": {
        "architect": {
            "build": {
               "options": {
                    "scripts": [
                         "node_modules/plotly.js/dist/plotly.min.js"
                    ]
                }
            }
        }
    }
}
```
in my angular.json

**Comment by andrefarzat** [comment]
Hello, @Simbaclaws , I believe the [`PlotlyViaWindowModule`](https://github.com/plotly/angular-plotly.js#plotly-via-window-module) suits better what you want to achieve. May I ask you to try it?

**Comment by Simbaclaws** [comment]
Dear @andrefarzat,

Apologies, apparently my code wasn't being pushed to the devices because there was an error in my console that I didn't see about maximum bundle size when compiling. It was giving me a fatal error.
Since I added plotly.min.js to my bundle it tells me it exceeds the maximum bundle size in angular.json.

I resolved this by upgrading the limit to roughly 10MB, this allowed me to successfully compile. Which updated the source code from the app. And plotly is now working correctly offline.

I was previously using PlotlyViaCDNModule in my code and didn't check whether the code on the mobile device had actually changed to the code I have written in my first post.

This issue can be closed.
